lib/core/src/lib/datatable/data-column/data-column.component.ts

```diff
@@ -61,6 +61,10 @@ export class DataColumnComponent implements OnInit {
     @Input()
     draggable: boolean = false;
 
+    /** Enable resize for column */
+    @Input()
+    resizable: boolean = true;
```

Comment:
redundant type because you assign true inline

Reply:
Removed